index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+
+<body>
```

Comment:
- [ ] Kindly fix the spacing between your header and the headline section to match the figma design

figma design 👇🏽 
![image](https://github.com/agneta1991/Portfolio/assets/63473584/a004a062-fd31-4c24-b226-6c06cb1ea7c4)

your design 👇🏽 
![image](https://github.com/agneta1991/Portfolio/assets/63473584/56de8eae-6029-45de-8575-8412f0ef80e9)